docs/usage/mend-hosted/hosted-apps-config.md

```diff
@@ -0,0 +1,79 @@
+## Mend Renovate App Configuration
```

Comment:
Do these settings only to github.com and not Bitbucket Cloud?

Reply:
Both. I will update the heading to "Mend-hosted Apps Configuration"